Makefile

```diff
@@ -91,6 +89,8 @@ verify: ## Verify code. Includes codegen, docgen, dependencies, linting, formatt
 	hack/validation/labels.sh
 	hack/validation/resources.sh
 	hack/validation/status.sh
+	cp -r pkg/apis/crds kwok/charts
```

Comment:
Consider doing a symlink like we do elsewhere with our CRDs

Reply:
I think the point is we don't want to modify the original CRDs when applying the additional kwok validation requirements

Reply:
That's exactly the point. i've had too many PRs where `make apply` made the change, but then I had to restore the change so it wouldn't be included. I'd liken this model to the same way that we do it in the AWS Provider. We essentially copy over the upstream nodepool/nodeclaim apis and add our own validations then apply it. KWOK is doing the same but just in the same repo.